### PR TITLE
story US785459 [next] Validate @zowe/cli@next installation works on z/OS USS and zLinux

### PR DESCRIPTION
**Pull request by JamesBauman**
Please review:

https://github.com/zowe/docs-site/blob/jabauman-USS-zLinux/docs/user-guide/cli-install-cli-next.md
https://github.com/zowe/docs-site/blob/jabauman-USS-zLinux/docs/user-guide/cli-using-daemon-mode.md
https://github.com/zowe/docs-site/blob/jabauman-USS-zLinux/docs/user-guide/cli-configure-daemon-on-zlinux-os.md
https://github.com/zowe/docs-site/blob/jabauman-USS-zLinux/docs/user-guide/cli-configure-scs-on-zlinux-os.md
https://github.com/zowe/docs-site/blob/jabauman-USS-zLinux/docs/user-guide/cli-configure-os-where-scs-unavailable.md

thx,

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

**Review by awharn — APPROVED** [review]
These changes look good to me. The `Configure daemon mode on z/Linux operating systems` section's list under step 3 has a bit of an odd spacing, but I don't see anything wrong in the markdown to cause that, aside from perhaps an extra line.

**Comment by JamesBauman** [comment]
@MikeBauerCA   pending your review.   

FYI that we added three articles to the doc site;  They are highlighted below

image


thx

File: docs/user-guide/cli-install-cli-next.md [hunk 1]
```diff
@@ -4,7 +4,23 @@
...
-***Prerequisites***
+## Installation guidelines
+  
+To install the CLI @next version on **Windows**, **Mac**, and **Linux** operating systems, follow the steps in [Install Zowe CLI from npm](install-zowe-cli-from-npm) or [Install Zowe CLI from a download](#install-zowe-cli-from-a-download).
+
+However, to install Zowe CLI @next on **z/Linux**, **z/OS UNIX System Services (USS)**, or on an operating system where the *Secure Credential Store* is *not required* or *cannot be installed*, use the following installation guidelines:
+
+*  To install Zowe CLI @next on a z/Linux operating system and you **require** the Secure Credential Store:
+   1. Follow the steps in [Configure Secure Credential Store on z/Linux operating systems](cli-configure-scs-on-zlinux-os).
+   2. Follow the steps in follow the steps in [Install Zowe CLI from npm](install-zowe-cli-from-npm) or [Install Zowe CLI from a download](#install-zowe-cli-from-a-download).
```
Comment (MikeBauerCA) [inline docs/user-guide/cli-install-cli-next.md:15]: "Follow the steps in" is written twice

File: docs/user-guide/cli-install-cli-next.md [hunk 2]
```diff
@@ -4,7 +4,23 @@
...
+*  To install Zowe CLI @next on a z/Linux operating system and you **do not require** the Secure Credential Store:
+   1. Follow the steps in follow the steps in [Install Zowe CLI from npm](install-zowe-cli-from-npm) or [Install Zowe CLI from a download](#install-zowe-cli-from-a-download).
+   2. Follow the steps in [Configure operating systems where the Secure Credential Store is not available](cli-configure-os-where-scs-unavailable).
+*  To install Zowe CLI @next on a USS system or on an operating system where you cannot install the Secure Credential Store:
+   1. Follow the steps in follow the steps in [Install Zowe CLI from npm](install-zowe-cli-from-npm) or [Install Zowe CLI from a download](#install-zowe-cli-from-a-download).
```
Comment (MikeBauerCA) [inline docs/user-guide/cli-install-cli-next.md:20]: "Follow the steps in" is written twice

**Review by MikeBauerCA — CHANGES_REQUESTED** [review]

File: docs/user-guide/cli-configure-scs-on-zlinux-os.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Install the following Linux packages on the z/Linux system:
+    - make
+    - gcc-c++ (sometimes available as g++)
+    - gnome-keyring
+    - libsecret (sometimes available as libsecret-1-0)
+    - libsecret-devel (sometimes available as libsecret-1-dev)
+    - Python 3.6 or later
+
+    **Note:** The z/Linux system where you are configuring SCS might require internet access to install the packages.
```
[the remark this answers is not in the document]
Comment (awharn) [reply docs/user-guide/cli-configure-scs-on-zlinux-os.md:17]: That makes sense. In this case, internet access will be required if you are using the Red Hat package repository. 

Some sites might host their own package repositories, but those repositories might not have all of the packages we require. In that case, someone will need to either add the required package(s) to that repository (I have no idea how you would go about doing that), or need internet access to use Red Hat's.

File: docs/user-guide/cli-configure-os-where-scs-unavailable.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Configure operating systems where the Secure Credential Store is not available
```
[the remark this answers is not in the document]
Comment (JamesBauman) [reply docs/user-guide/cli-configure-os-where-scs-unavailable.md:1]: done

File: docs/user-guide/cli-configure-os-where-scs-unavailable.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    ```
+    "CredentialManager": false
+    ```
+3. Save the changes.
+
+## Team Configuration
+
+Team configuration is stored in `zowe.config.json`.
+
+Team configuration can be created without access to the Secure Credential Store. However, team configuration does not store sensitive user information on the system. Subsequent commands prompt for the user’s sensitive information when it not provided on the command line, and will attempt to save it with the new Auto Store functionality. Users may experience errors when Auto Store cannot save sensitive information securely. To mitigate this error, disable the Auto Store functionality by modifying the `zowe.config.json` or `zowe.config.user.json` file and changing the autoStore property from true to false.
```
[the remark this answers is not in the document]
Comment (JamesBauman) [reply docs/user-guide/cli-configure-os-where-scs-unavailable.md:20]: done

File: docs/user-guide/cli-configure-scs-on-zlinux-os.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (JamesBauman) [reply docs/user-guide/cli-configure-scs-on-zlinux-os.md:17]: done, please review.....

File: docs/user-guide/cli-configure-scs-on-zlinux-os.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+2. If you are configuring SCS on a Ubuntu z/Linux operating system, no further action is required. You can now install Zowe CLI. For all other platforms (RHEL), continue to the next step.
+
+3. Enable the `rhel-#-for-system-z-optional-rpms` repository to download libsecret-devel. If your license entitles you to this repository, enable the repository by issuing the following command:
+
+    ```
+    subscription-manager repos —-enable rhel-#-for-system-z-optional-rpms
+    ```
+
+    - **#**: Replace # with the major version of RHEL that is running on the z/Linux system.
```
[the remark this answers is not in the document]
Comment (JamesBauman) [reply docs/user-guide/cli-configure-scs-on-zlinux-os.md:27]: done

File: docs/user-guide/cli-configure-scs-on-zlinux-os.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+4. If you are configuring SCS to run on RHEL V8.x or later, no further action is required. You can now install Zowe CLI. For RHEL V7.x, continue to the next step.
+5. Install the Red Hat Developer Toolset to ensure that you are running version of the gcc-c++ compiler that can build the SCS native binaries.
```
[the remark this answers is not in the document]
Comment (JamesBauman) [reply docs/user-guide/cli-configure-scs-on-zlinux-os.md:29]: doh!  

done

File: docs/user-guide/cli-install-cli-next.md (hunk 1 above)
Comment (JamesBauman) [reply docs/user-guide/cli-install-cli-next.md:15]: doh!
doh!
doh!

Done-done-done...

File: docs/user-guide/cli-install-cli-next.md (hunk 2 above)
Comment (JamesBauman) [reply docs/user-guide/cli-install-cli-next.md:20]: done

File: docs/user-guide/cli-configure-scs-on-zlinux-os.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (JamesBauman) [reply docs/user-guide/cli-configure-scs-on-zlinux-os.md]: @awharn and @MikeBauerCA 

Please review the words the I used for this change/

**Comment by JamesBauman** [comment]
note the following URL change (changed topic title)
**From**
https://github.com/zowe/docs-site/blob/jabauman-USS-zLinux/docs/user-guide/cli-configure-os-where-scs-unavailable.md
**To**
https://github.com/zowe/docs-site/blob/jabauman-USS-zLinux/docs/user-guide/cli-configure-cli-on-os-where-scs-unavailable.md

File: docs/user-guide/cli-configure-scs-on-zlinux-os.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Install the following Linux packages on the z/Linux system:
+    - make
+    - gcc-c++ (sometimes available as g++)
+    - gnome-keyring
+    - libsecret (sometimes available as libsecret-1-0)
+    - libsecret-devel (sometimes available as libsecret-1-dev)
+    - Python 3.6 or later
+
+    **Note:** If you are installing the Linux packages on a RHEL system, the system where you are configuring SCS might require Internet access. When a site hosts its own package repositories, the repositories might not contain all of the packages that are required to configure the SCS. Under this scenario, the RHEL system requires Internet access to install the required packages.
```
Comment (awharn) [inline docs/user-guide/cli-configure-scs-on-zlinux-os.md:17]: This isn't specific to a RHEL system, if you are installing Linux packages, any system might require internet access.

**Review by awharn — CHANGES_REQUESTED** [review]
I think there is just one small change that needs to be made.

File: docs/user-guide/cli-configure-os-where-scs-unavailable.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (JamesBauman) [reply docs/user-guide/cli-configure-os-where-scs-unavailable.md]: This was updated as requested...


https://github.com/zowe/docs-site/blob/jabauman-USS-zLinux/docs/user-guide/cli-configure-cli-on-os-where-scs-unavailable.md

Dismissing the review

**Review by MikeBauerCA — APPROVED** [review]

**Comment by JamesBauman** [comment]
Signed-off-by: JamesBauman <james.bauman@broadcom.com>

**Pull request merged**